diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0bdb7088..2e7a5cca 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -101,7 +101,7 @@ if(EMSCRIPTEN)
     add_compile_options( $<IF:$<CONFIG:Debug>,-O0,-O3> )
     add_link_options( $<IF:$<CONFIG:Debug>,-gsource-map,-O3> )
 elseif(MSVC)
-    add_compile_options( $<IF:$<CONFIG:Debug>,/Gz,/O2> )
+    add_compile_options( $<IF:$<CONFIG:Debug>,,/O2> )
 else()
     add_compile_options( $<IF:$<CONFIG:Debug>,-O0,-O3> )
     add_link_options( $<IF:$<CONFIG:Debug>,-g,-O3> )

